src/views/vehicleServiceDetails/vehicleServicepage.jsx

```diff
@@ -480,26 +582,40 @@ class vehicleServicepage extends Component {
               </button>
             </div>
             <div className="modal-body">
-              <AvForm onValidSubmit={this.updateVehicleServideDetails}>
+              <AvForm
+                data-update_vehi_serv_method="update_date"
+                onValidSubmit={this.updateVehicleServideDetails}
+              >
                 <AvField
                   name="requestedServiceDate"
                   label="Requested Service Date"
-                  type="date"
+                  type="dateTime"
+                  validate={{ format: 'DD/MM/YYYY HH:MM:SS' }}
+                  placeholder={'DD/MM/YYYY HH:MM:SS'}
                   required
                   onChange={this.formDataChange}
                   className="blue_lable"
                   value={this.state.requestedServiceDate}
                   selected={this.state.requestedServiceDate}
-                  placeholder="Requested Service Date"
                 />
                 <center>
                   <Button color="success">Update</Button>
                 </center>
               </AvForm>
-              <center>
-                <hr />
-                <Button color="success">Complete Schedule </Button>
-              </center>
+              {this.state.status !== 'On Due' ? (
```

Comment:
destructure state on render method

status should be single word seperated by _ and uppercase.
add status label as we wanna display.

If client requests to change the words On Due to something else. changes will be in database level